Source/WebCore/platform/audio/AudioSession.cpp

```diff
@@ -47,19 +47,35 @@ namespace WebCore {
 WTF_MAKE_TZONE_ALLOCATED_IMPL(AudioSession);
 
 bool AudioSession::s_shouldManageAudioSessionCategory { false };
+static bool s_mediaPlaybackEnabled { false };
 
 static std::optional<UniqueRef<AudioSession>>& sharedAudioSession()
 {
     static NeverDestroyed<std::optional<UniqueRef<AudioSession>>> session;
     return session.get();
 }
 
+static std::optional<UniqueRef<AudioSession>>& dummyAudioSession()
+{
+    static NeverDestroyed<std::optional<UniqueRef<AudioSession>>> dummySession;
+    return dummySession.get();
+}
+
```

Comment:
This is an unusual pattern because this and `AudioSession::createDummy` are both used in only one place. 

Why not combine the two:

```suggestion
static std::optional<UniqueRef<AudioSession>>& dummyAudioSession()
{
    static NeverDestroyed<std::optional<UniqueRef<AudioSession>>> dummySession = makeUniqueRef<AudioSession>();
    return dummySession.get();
}
```